loader/loader.c

```diff
@@ -2969,19 +2969,13 @@ static VkResult read_data_files_in_search_paths(const struct loader_instance *in
     if (path_override != NULL) {
         override_path = path_override;
     } else if (env_override != NULL) {
-        // Don't allow setuid apps to use the env var
-        if (is_high_integrity()) {
-            loader_log(inst, VULKAN_LOADER_WARN_BIT, 0,
-                       "read_data_files_in_search_paths: Ignoring override %s due to high-integrity", env_override);
-        } else {
-            override_env = loader_secure_getenv(env_override, inst);
+        override_env = loader_secure_getenv(env_override, inst);
```

Comment:
We discussed this offline, but wanted to document that the check for `is_high_integrity()` here is redundant since `loader_secure_getenv()` calls that function internally.